### ISSUE TITLE
PG1 - Undefined index errors occurring on field_page_image and field_page_caption

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#440]
Errors occurring on basic page.

```
Notice: Undefined index: field_page_image in ug_theme_preprocess_views_view_fields__pg1() (line 176 of /var/www/html/aoda.client.uoguelph.ca/aoda/profiles/ug/themes/ug/ug_theme/template.php).
Notice: Trying to get property of non-object in ug_theme_preprocess_views_view_fields__pg1() (line 176 of /var/www/html/aoda.client.uoguelph.ca/aoda/profiles/ug/themes/ug/ug_theme/template.php).
Notice: Undefined index: field_page_caption in ug_theme_preprocess_views_view_fields__pg1() (line 177 of /var/www/html/aoda.client.uoguelph.ca/aoda/profiles/ug/themes/ug/ug_theme/template.php).
Notice: Trying to get property of non-object in ug_theme_preprocess_views_view_fields__pg1() (line 177 of /var/www/html/aoda.client.uoguelph.ca/aoda/profiles/ug/themes/ug/ug_theme/template.php).
```

Issue appears to be related to the template.php file. Image and caption fields were added as part of the basic page fields, but they do not appear to exist on the view. Unsure why the warning is only showing up now for the first time.

Recommend either 

(a) adding the image and caption fields to the Basic Page detail page view (PG1) as per request https://github.com/ccswbs/hjckrrh/issues/400 or 
(b) removing the field styling in template.php until the image and caption fields can be added

**Comment by brharp** [comment]
@mmafe Do you remember on which site you saw this error?

**Comment by mmafe** [comment]
@brharp Start Online, however, the error has been fixed on that particular site. It only shows up if someone uses the PG1 pane (which they're not supposed to use).

**Comment by brharp** [comment]
@mmafe I'm wondering if we even need that pane. The page manager variant for page nodes doesn't use it, and I can't think of another obvious use case for it.

**Comment by mmafe** [comment]
@brharp I don't think we need it at this point.

**Comment by brharp** [comment]
Fixed in 7.4.0
